info/v1/machine.go

```diff
@@ -136,7 +136,7 @@ type MachineInfo struct {
 	CpuFrequency uint64 `json:"cpu_frequency_khz"`
 
 	// The amount of memory (in bytes) in this machine
-	MemoryCapacity int64 `json:"memory_capacity"`
+	MemoryCapacity uint64 `json:"memory_capacity"`
```

Comment:
This change makes sense, but it is a breaking API change :( Its not clear if it will break any users though. So let's try rolling this out and then revert if required.